src/style.css

```diff
@@ -0,0 +1,75 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 800;
+}
+
+.form {
+  width: 100%;
+  position: relative;
+  margin-top: 20px;
+}
+
+.searchBar {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  color: #3d4e61;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.searchBar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.searchBar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-weight: 800;
+  outline: none;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.searchBar:focus::placeholder {
+  font-weight: 300;
+  text-shadow: none;
+}
+
+.SearchBar--small {
```

Comment:
Be consistent with naming classes - if you are using camelCase use it everywhere :)